relnotes/typedefBase.feature.md

```diff
@@ -0,0 +1,4 @@
+* `ocean.meta.types.Typedef`
+
+  New helper function `typedefBase` casts value of type created
```

Comment:
Some articles would help here: "The new helper function `typedefBase` casts a value of a type created..."